spec/cc/engine/analyzers/php/main_spec.rb

```diff
@@ -2,12 +2,17 @@
 require 'cc/engine/analyzers/php/main'
 require 'cc/engine/analyzers/reporter'
 require 'cc/engine/analyzers/engine_config'
+require 'cc/engine/analyzers/file_list'
 require 'flay'
 require 'tmpdir'
 
 module CC::Engine::Analyzers::Php
   describe Main do
-    before { @code = Dir.mktmpdir }
+    before do
+      @code = Dir.mktmpdir
+      Dir.chdir(@code)
```

Comment:
Well, you can simulate `around` with `before` & `after`, and `chdir` back to the correct cwd after the run, right?

I'm thinking about things like code coverage we might eventually integrate, randomized test order (which I wish minitest did out of the box), etc. Basically, unexpected working directory is one of those things that will eventually bite you, and is a pain to debug when it does. We should avoid it if we can.